docs/deploy.md

```diff
@@ -0,0 +1,24 @@
+# Deploying
+
+If you would like to run your own instance of this plugin, see the [docs for deploying probot plugins](https://github.com/probot/probot/blob/master/docs/deployment.md).
+
+## Setup
+
+```
+# Install dependencies
+npm install
+
+# Run the bot
+npm start
+```
+
+Teacherbot requires these **Permissions & events** for the GitHub App:
+
+- Administration - **Read & Write**
+- Issues - **Read & Write**
```

Comment:
@crichID the [permissions](https://github.com/organizations/githubschool/settings/apps/teacherbot-staging/permissions) that teacherbot requires are:

- Repository Administration - **Read & Write** - to add collaborators that comment on new issues
- Issues - **Read & Write** - to communicate with issue openers
- Pull Requests - **Read & Write** - to remind users to delete their branch upon merging or closing
- Single file - **Read-only** - to read the teacherbot configuration file